src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java

```diff
@@ -1238,6 +1238,8 @@ public class CaseData extends Et1CaseData {
     private String respondNotificationWhoRespond;
     @JsonProperty("respondNotificationCaseManagementMadeBy")
     private String respondNotificationCaseManagementMadeBy;
+    @JsonProperty("respondNotificationRequestMadeBy")
```

Comment:
I wonder why do you need this field here? You can access it any way from SendNotificationType.respondNotificationTypeCollection.RespondNotificationType?

Reply:
its needed on the case data for the RespondNotification event. 
When respondNotification is saved it is added to the respondNotificationTypeCollection